docs/setup/install/targz.asciidoc

```diff
@@ -125,12 +125,6 @@ important data later on.
   | $KIBANA_HOME\data
  d|
 
-| optimize
```

Comment:
Given that "delete the contents of the optimize directory" is pretty common advice, we should probably keep its location explicitly documented, even if it's `$KIBANA_HOME\data\optimize`